.prettierrc

```diff
@@ -0,0 +1,9 @@
+{
+  "semi": false,
+  "singleQuote": true,
+  "bracketSpacing": true,
+  "trailingComma": "es5",
+  "useTabs": false,
+  "tabWidth": 2,
+  "printWidth": 80
```

Comment:
We could bump this up to `100` or `120`
`printWidth` of `80` is coming from a time where 800x600 monitors was common...
`120` is a little bit to wide for side-by-side reviews in the Web-GUI, but it's fine because not every line is so long
`110` is near the side-by-side view
so `100` is a safe pick

Personally I prefer `120`

Reply:
After our discussion, we will keep 80 then.